### ISSUE TITLE
Cooking Pot bug/crash

### ISSUE THREAD

**Issue by Eilisti** [LeandroTheDev/level_up#4]
When you're done cooking with the cooking pot on the firepit, if you take it and place it back in the firepit, it'll crash your game. Before it crashes, it'll utilize all of your CPU power for around 30 seconds, causing everything to be really slow.

**Comment by Eilisti** [comment]
I should mention that this issue only occurs in version 1.1.3

**Comment by LeandroTheDev** [comment]
can you provide me the logs in the server with enableExtendedLog option?

**Comment by Eilisti** [comment]
[client-audit.txt](https://github.com/LeandroTheDev/level_up/files/15027898/client-audit.txt)
[client-chat.txt](https://github.com/LeandroTheDev/level_up/files/15027899/client-chat.txt)
[client-debug.txt](https://github.com/LeandroTheDev/level_up/files/15027900/client-debug.txt)
[client-main.txt](https://github.com/LeandroTheDev/level_up/files/15027901/client-main.txt)
[server-audit.txt](https://github.com/LeandroTheDev/level_up/files/15027902/server-audit.txt)
[server-build.txt](https://github.com/LeandroTheDev/level_up/files/15027903/server-build.txt)
[server-chat.txt](https://github.com/LeandroTheDev/level_up/files/15027904/server-chat.txt)
[server-debug.txt](https://github.com/LeandroTheDev/level_up/files/15027905/server-debug.txt)
[server-event.txt](https://github.com/LeandroTheDev/level_up/files/15027906/server-event.txt)
[server-main.txt](https://github.com/LeandroTheDev/level_up/files/15027907/server-main.txt)
[server-storyevent.txt](https://github.com/LeandroTheDev/level_up/files/15027908/server-storyevent.txt)
[server-worldgen.txt](https://github.com/LeandroTheDev/level_up/files/15027909/server-worldgen.txt)

**Comment by Eilisti** [comment]
So far, the crash has occurred when cooking Puffballs in two quantities and red meat in the same quantities; I'm not sure if other ingredients will also cause a crash.

**Comment by Eilisti** [comment]
Also, the crash only occurs if you cook the meal through and then put the pot back to the left square after it has been cooked all the way.

**Comment by LeandroTheDev** [comment]
Thank you for the report, i finded the problem, it will be fixed in 1.1.4

What i need to fix that is to make a check for reheating foods to not call the experience function again (a lot of times) causing the cpu overflow.
```css
if (Configuration.expMultiplySingleCooking.TryGetValue(__instance.inputStack.Collectible.Code.ToString(), out double _)) return;
else if (Configuration.expMultiplyPotsCooking.TryGetValue(__instance.inputStack.Collectible.Code.ToString(), out double _)) return;
```
